docs/docs/axes/styling.md

```diff
@@ -55,3 +46,68 @@ The majorTick configuration is nested under the ticks configuration in the `majo
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | `enabled` | `boolean` | `false` | If true, major ticks are generated. A major tick will affect autoskipping and `major` will be defined on ticks in the scriptable options context.
+
+## Indexable Options
```

Comment:
Maybe we should just refer to `general/options.md` here instead of duplicating this part?